### ISSUE TITLE
Manually-added Non-Steam games that use Proton show 'Missing Game Files' just like missing Steam-installed games

### ISSUE THREAD

**Issue by OldWarez** [scawp/Steam-Deck.Shader-Cache-Killer#3]
Things like ProtonTricks show non-Steam names properly so there might be a simple way to match them with the random-numbered shader and compatdata folders

**Comment by scawp** [comment]
I had a look at ProtonTricks - https://github.com/Matoking/protontricks/blob/e59e0e70e3a26158cd793ee7c482fdb40c7cc019/src/protontricks/steam.py#L1019
and basically it uses Python libraries that aren't on the Deck so would require a flatpak. Instead I've added some code to check a couple of logs that might contain the id and names so at least some non-steam games should show now.

**Comment by SaadAlareqi** [comment]
> I had a look at ProtonTricks - https://github.com/Matoking/protontricks/blob/e59e0e70e3a26158cd793ee7c482fdb40c7cc019/src/protontricks/steam.py#L1019 and basically it uses Python libraries that aren't on the Deck so would require a flatpak. Instead I've added some code to check a couple of logs that might contain the id and names so at least some non-steam games should show now.

Yes i noticed that too. i was surprised that nearly all my games shows as "Missing Game Files" while in ProtonTricks they show up correctly. i was going to add a ticket here but i saw this. i think this should be a high priority as it is useless at this state and it just needs to implement what ProtonTrcks does. you could create a new app for this purpose and call it like Games Uninstaller since this functionality is different than Shader cache.
thanks for the good work

**Comment by blueplanet88** [comment]
> I had a look at ProtonTricks - https://github.com/Matoking/protontricks/blob/e59e0e70e3a26158cd793ee7c482fdb40c7cc019/src/protontricks/steam.py#L1019 and basically it uses Python libraries that aren't on the Deck so would require a flatpak. Instead I've added some code to check a couple of logs that might contain the id and names so at least some non-steam games should show now.

Any further progress on this? There's still no way to manage compatdata for non-steam games
